docs/source/tutorials-basic/driving.rst

```diff
@@ -0,0 +1,197 @@
+.. _tutorials-basic-driving-chapter:
+
+**************************
+Driving Around
+**************************
+
+Driving the GoPiGo3 around is only a matter of choosing the right methods from the :py:class:`~easygopigo3.EasyGoPiGo3` class.
+In all of our examples, the :py:mod:`~easygopigo3` is required and needs to be imported and a :py:class:`~easygopigo3.EasyGoPiGo3` object
+has to be instantiated this way:
+
+.. code-block:: python
+
+   from easygopigo3 import EasyGoPiGo3 # importing the EasyGoPiGo3 class
+   gpg = EasyGoPiGo3() # instantiating a EasyGoPiGo3 object
+
+Instantiating :py:class:`~easygopigo3.EasyGoPiGo3` object can fail if the GoPiGo3 is not present.
+Check the :py:class:`~easygopigo3.EasyGoPiGo3.__init__` constructor of this class to see what exceptions can be raised.
+
+=======================
+Forward, R/L, Backward
+=======================
+
+The most basic commands don't offer control over how much distance the GoPiGo3 travels. They only get called once and then
+it's the responsability of the user to stop the GoPiGo3 from moving, regularly by using the :py:meth:`~easygopigo3.EasyGoPiGo3.stop` command.
+
+We first need to import the :py:class:`easygopigo3` and the :py:mod:`python:time` modules and instantiate a EasyGoPiGo3 object.
+
+.. code-block:: python
+
+    # import the time library for the sleep function
+    import time
+    import easygopigo3 as easy
```

Comment:
You're not importing and instantiating with the same approach as the one you listed at the top of the tutorial. Pick one stick to it. Personally I'm partial to the one here over the one at the top.

Reply:
Done.